src/Home.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {HotelsTable} from "./hotel/components/HotelsTable";
+import {CitiesTable} from "./city/components/CitiesTable";
+import {CountriesTable} from "./country/components/CountriesTable";
+
+export const Home = () => {
+    return(
+        <>
+            <HotelsTable />
```

Comment:
Do you use <br> tag for the padding or margin purposes?

Reply:
Do you mean <br /> tag ? For margin between tables.

Reply:
Do you use
tag for the padding or margin purposes?